### DISCUSSION TITLE
User docs?

### DISCUSSION THREAD

**Discussion by davelab6** [fontra/fontra#587] [General]
What's the plan, forecast, schedule, for user docs, so that a random pro level type designer can get started using Fontra?

**Comment by justvanrossum** [comment]
Good questions! We will draft a plan and will get back to you.

We also don't have a good definition of what could be considered "feature-complete", but I believe the answer to that is at least somewhat related to the user docs question.

**Comment by ollimeier** [comment]
@davelab6 There is a Fontra-Docs page initially created by @gferreira : 
https://gferreira.github.io/fontra-docs/

I recently started to extend and update it with a view pull requests:
https://github.com/gferreira/fontra-docs/pulls

Long term, we should consider to move it to googlefonts-github. 
But for now, it's a great starting point.

A link to this page has been added to the menu bar, already. As you can see in the following screenshot:
<img width="1485" alt="Screenshot 2024-03-01 at 10 26 59" src="https://github.com/googlefonts/fontra/assets/21055547/432a6a26-1742-4082-96e8-a57ab9568863">

It will be an ongoing project. So, I am not sure if you want to close this discussion?

**Reply by ollimeier** [reply]
@justvanrossum and @davelab6 
Probably the right moment to move this to a different location, please see: https://github.com/gferreira/fontra-docs/pull/12#issuecomment-1972882062

**Comment by ollimeier** [comment]
@justvanrossum @JeremieHornus Is there any progress on moving the documentation to a different location? I would like to update the documentation every now and then (when ever it makes sense), eg. when we introduce a new feature like we just did this week with the new shape tool.

**Comment by ollimeier** [comment]
I am closing this discussion because:
1. it's an ongoing project (we will take of it)
2. the github repository has been moved from @gferreira to googlefonts successfully
